### ISSUE TITLE
Not an issue - thank you.

### ISSUE THREAD

**Issue by srkaeppler** [mdo/ama#277]
Hi Mark-

Thanks for developing bootstrap, I am hoping to use it for developing a laboratory website.  I'm excited to use it.

Second and more obliquely, glad to see someone who went to Friess Lake went on to do great things!

All the best,
Steve

**Comment by mdo** [comment]
> Thanks for developing bootstrap, I am hoping to use it for developing a laboratory website. I'm excited to use it.

You're very welcome, and that's so awesome to hear!

> Second and more obliquely, glad to see someone who went to Friess Lake went on to do great things!

Whoa! Crazy small world. I'm sorry I don't recognize the name, when did you attend/graduate?

**Comment by srkaeppler** [comment]
Hi Mark, alright you confirmed it!  Chances are you might remember my brother - I think he was in the grade above you and I remember your brother Brad, he was in the grade above me.  I started to look into bootstrap and I saw "mark otto", so then I googled you(just out of curiosity) and was like "is that Brad's younger brother?"  Anyway, sorry for sounding like a stalker...  Graduated HUHS 2001, Friess Lake 1997.  Also, I hope you're liking the Bay Area, my family and I just left the area in December 2017 - we were living/working Menlo Park (and not for facebook)...
